.env.example

```diff
@@ -122,7 +122,10 @@ BASEURL=https://localhost:3015/
 # Get your CoinMarketCap credentials please check the social network configuration file in the docs folder of this project for more information.
 CMR_URL=YOUR_CMR_URL
 CMCAPIKEY=YOUR_COINMARKETCAP_KEY
-
+CMC_GLOBL_URL=https://pro-api.coinmarketcap.com/v1/global-metrics/quotes/latest
+CMC_HISTORY_URL=https://pro-api.coinmarketcap.com/v3/cryptocurrency/quotes/historical
+CMC_URl=https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest
+CMC_CRYPTO_URL=https://pro-api.coinmarketcap.com/v1/cryptocurrency/quotes/latest
```

Comment:
Why not use version v3 of CoinMarketCap? Using two different versions for the same API could create availability issues in the future.

Reply:
the upgrade to v3 is required to obtain more details, we can also upgrade to version 2 or 3 but we need to be sure not to damage all the work we have done, we can do this in another task.